modules/storages/app/components/storages/project_storages/members/row_component.rb

```diff
@@ -100,7 +100,7 @@ def storage_connection_status
 
     def oauth_client_connected?
       storage.oauth_client.present? &&
-        member.principal.remote_identities.exists?(oauth_client: storage.oauth_client)
+        member.principal.remote_identities.exists?(auth_source: storage.oauth_client)
```

Comment:
I don't quite understand where this method is used (yet). I guess there is an overview showing who in a project has access to a storage?

This will not yet properly work for SSO users. Would you agree? I am happy to have this extracted into a bug/feature, but we should make this visible as to not forget it, unless we fix it immediately.

Reply:
Yes, it does not work properly for SSO users at the moment.
The method is used on project storage members page. The page is used to give a members overview which show who have completed OAuth flow and who have not. The page can be found here:  https://openproject.local/projects/ppproject/settings/project_storages/6/members.
I filed a bug: https://community.openproject.org/projects/cross-application-user-integration-stream/work_packages/61880/activity